src/app/shared/layout/footer/footer.component.html

```diff
@@ -0,0 +1,185 @@
+<ecl-site-footer>
```

Comment:
Now, it makes sense in theory to wrap the main components of the page as we do for all the others, but based on the application we are using, there are "functionalities" related to these components that are currently working in the eUI app.component, not outside of it.
For instance the footer has translations already available (a few ones in french), now in this pr we lose the language switcher so this cannot even be tested.